au4/src/au3wrap/internal/au3audiodevicesmanager.h

```diff
@@ -6,16 +6,30 @@
 #define AU_AU3WRAP_AU3AUDIODEVICESMANAGER_H
 
 #include <wx/arrstr.h>
+#include "global/types/string.h"
 
 #include "libraries/lib-utility/IteratorX.h"
 #include "libraries/lib-strings/wxArrayStringEx.h"
+#include "playback/iaudiodevicesmanager.h"
 
 namespace au::au3 {
-class Au3AudioDevicesManager
+class Au3AudioDevicesManager : public playback::IAudioDevicesManager
 {
 public:
     void init();
 
+    muse::StringList audioOutputDevices() const override;
+    muse::String currentAudioOutputDevice() const override;
+    void setAudioOutputDevice(const muse::String& deviceName) override;
+    muse::async::Notification audioOutputDeviceChanged() const override;
+
+    void handleDeviceChange() override;
+
+    muse::StringList audioApi() const override;
+    int currentAudioApiIndex() const override;
+    void setAudioApi(int audioApiIndex) override;
```

Comment:
I know that it's done similar to MuseScore... But it looks so-so... It would be more appropriate to have something like:
```
muse::String currentAudioApi() const override;
void setAudioApi(const muse::String& audioApi) override;
```